### ISSUE TITLE
Opening nested tabs via url/hash

### ISSUE THREAD

**Issue by birdmedia** [wangaz/contao-jquery-ui-tabs#13]
Is there a way to open nested tabs via url/hash? As far as I can see, only the first level tabs can be opened directly via url/hash. The nested tabs are addressed correctly but there is no mechanism to open the parent tabs automatically.

**Comment by 7FyOh** [comment]
No, so far, this functionality is not implemented :)

**Comment by birdmedia** [comment]
Is it possible to add this feature? :)

**Comment by 7FyOh** [comment]
Yes, that should be doable but I cannot promise any date of completion as we are well occupied at the moment. Apart from that you are also welcome to submit a Pull request containing that feature :)
